src/components/lastUpdated/LastUpdatedActions.ts

```diff
@@ -0,0 +1,38 @@
+import { IReduxAction } from '../../utils/ReduxUtils';
+
+export const LastUpdatedActions = {
+  addLastUpdated: 'ADD_LAST_UPDATED',
+  removeLastUpdated: 'REMOVE_LAST_UPDATED',
+  changeLastUpdated: 'CHANGE_LAST_UPDATED'
+};
+
+export interface ILastUpdatedPayload {
+  id: string;
+}
+
+export const addLastUpdated = (id: string): IReduxAction<ILastUpdatedPayload> => {
+  return {
+    type: LastUpdatedActions.addLastUpdated,
```

Comment:
Parameters and the return type of the function are different for each actions AND it's kind of the Redux way to to it.